### PR TITLE
Actualize symfony files

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

**Comment by OskarStark** [comment]
looks good to me and this are all standard files, right?

@rande please have a look and merge this PR. Thank you!

**Comment by sagikazarmark** [comment]
Yup, I've just copied them from a fresh symfony install. I also altered some of the configuration files where the config was "the old" default.

File: web/app.php [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
+// You should change the ApcClassLoader first argument to a unique prefix
+// in order to prevent cache key conflicts with other applications
+// also using APC.
+/*
+$apcLoader = new ApcClassLoader(sha1(__FILE__), $loader);
+$loader->unregister();
+$apcLoader->register(true);
+*/
 
-$request = Sonata\PageBundle\Request\RequestFactory::createFromGlobals('host_with_path');
```
Comment (rande) [inline web/app.php]: You need to keep this line as it is require by PageBundle.

File: web/app_dev.php [hunk 2]
```diff
@@ -1,32 +1,31 @@
...
 if (isset($_SERVER['HTTP_CLIENT_IP'])
     || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
-    || !in_array(@$_SERVER['REMOTE_ADDR'], array('127.0.0.1', 'fe80::1', '::1'))
+    || !(in_array(@$_SERVER['REMOTE_ADDR'], array('127.0.0.1', 'fe80::1', '::1')) || php_sapi_name() === 'cli-server')
 ) {
     header('HTTP/1.0 403 Forbidden');
     exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
 }
 
-$request = Sonata\PageBundle\Request\RequestFactory::createFromGlobals('host_with_path');
```
Comment (rande) [inline web/app_dev.php]: You need to keep this line

Comment (sagikazarmark) [inline web/app_dev.php]: Sorry, overlooked this.

File: web/app.php (hunk 1 above)
Comment (sagikazarmark) [inline web/app.php]: Sorry, overlooked this.

**Comment by sagikazarmark** [comment]
Added those lines back.

File: web/app_dev.php [hunk 3]
```diff
@@ -1,32 +1,33 @@
...
+Debug::enable();
+
+require_once __DIR__.'/../app/AppKernel.php';
+
 $request = Sonata\PageBundle\Request\RequestFactory::createFromGlobals('host_with_path');
 
 $kernel = new AppKernel('dev', true);
-
+$kernel->loadClassCache();
+$request = Request::createFromGlobals();
```
Comment (rande) [inline web/app_dev.php]: So you need to remove this one ;)

File: web/app.php [hunk 4]
```diff
@@ -1,22 +1,32 @@
...
 
 $request = Sonata\PageBundle\Request\RequestFactory::createFromGlobals('host_with_path');
 
 $kernel = new AppKernel('prod', false);
+$kernel->loadClassCache();
+//$kernel = new AppCache($kernel);
 
+// When using the HttpCache, you need to call the method in your front controller instead of relying on the configuration parameter
+//Request::enableHttpMethodParameterOverride();
+$request = Request::createFromGlobals();
```
Comment (rande) [inline web/app.php]: So you need to remove this one ;)

Comment (sagikazarmark) [inline web/app.php]: Damn. ;)

**Comment by sagikazarmark** [comment]
I hope the ordering didn't matter. I simply replaced the original line with the custom one.

**Comment by OskarStark** [comment]
can you please squash your commits? @sagikazarmark

**Comment by sagikazarmark** [comment]
Done

**Comment by OskarStark** [comment]
Sorry for my late reply, but can you finally review your PR?

@core23 can you please have a look, i think we can merge this after @sagikazarmark checked it again

**Comment by core23** [comment]
@OskarStark I am not using the sandbox :/

**Comment by sagikazarmark** [comment]
LGTM

I will open a separate PR for updating the directory structure to Symfony 3.0

**Comment by OskarStark** [comment]
thank you @sagikazarmark

**Pull request merged**